src/Uno.UI/UI/Xaml/Controls/StackPanel/StackPanel.Layout.cs

```diff
@@ -115,6 +115,10 @@ protected override Size ArrangeOverride(Size arrangeSize)
 				snapPoints.RemoveAt(count);
 			}
 
+			var arrangedSize = isHorizontal
```

Comment:
Could you use a different variable name here - 'arrangeSize' and 'arrangedSize' are too similar for comfort